_tests/browser_tests/spec_suites/shared/shared_the-bureau.js

```diff
@@ -0,0 +1,19 @@
+TheBureauPage = require( '../../page_objects/page_the-bureau.js' );
```

Comment:
@anselmbradford This is purely for clarity and reducing mental load. Without the prefix, you would end up with two files named the same differentiated only by its location in the file system. We felt it would be less confusing to prefix it with 'page_', but everything is game for discussion.